Running FedStream with: Dir=1.0, Init_Class=10, Epoch=20
Files already downloaded and verified
Files already downloaded and verified
has read
Files already downloaded and verified
Files already downloaded and verified
Traceback (most recent call last):
  File "/home/kemove/lt/FedAlgorithm/alg/FedStream.py", line 770, in <module>
    alg.online_train()
  File "/home/kemove/lt/FedAlgorithm/alg/FedStream.py", line 733, in online_train
    global_loss_list, accuracy_list = self.alg.run(
                                      ^^^^^^^^^^^^^
  File "/home/kemove/lt/FedAlgorithm/alg/FedDyn.py", line 120, in run
    prev_grads_list = [
                      ^
  File "/home/kemove/lt/FedAlgorithm/alg/FedDyn.py", line 121, in <listcomp>
    self.init_prev_grads(self.net) for _ in range(self.num_client)
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/kemove/lt/FedAlgorithm/alg/FedDyn.py", line 96, in init_prev_grads
    prev_grads = torch.zeros_like(param.view(-1))
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
RuntimeError: CUDA error: out of memory
CUDA kernel errors might be asynchronously reported at some other API call, so the stacktrace below might be incorrect.
For debugging consider passing CUDA_LAUNCH_BLOCKING=1.
Compile with `TORCH_USE_CUDA_DSA` to enable device-side assertions.

